### DISCUSSION TITLE
Try GraphScope in your browser now !

### DISCUSSION THREAD

**Discussion by yecol** [alibaba/GraphScope#125] [Show and tell]
Provide [colab](https://colab.research.google.com/)-like notebook as playground.

Refs:
- https://research.google.com/colaboratory/local-runtimes.html
- https://github.com/googlecolab/colabtools/issues/266

**Comment by yecol** [comment]
maybe a  [jupyterhub](https://jupyterhub.readthedocs.io/en/latest/getting-started/index.html) + a k8s cluster on aliyun is a possbile solution.

**Comment by wenyuanyu** [comment]
@lidongze0629  Maybe Use JupyterHub with the new JupyterLab?

**Comment by lidongze0629** [comment]
https://github.com/jupyterhub/jupyterhub/issues/690#issuecomment-336445370
https://github.com/jupyterhub/jupyterhub-deploy-docker/pull/27/files


https://github.com/GraphScope/tutorials/commit/80ae7538eed711f299b1e39476c038d978d4189b

**Comment by lidongze0629** [comment]
Here already deploy a JupyterLab service where you can have a try in your browser:  https://try.graphscope.app
